### PR TITLE
Add Actframework benchmark set

### PR DESCRIPTION
**Pull request by greenlaw110**

### CODE REVIEW

File: frameworks/Java/act/pom.xml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <version>1.0.0-SNAPSHOT</version>
+    <name>ActFramework sample project</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <act.version>0.3.1-SNAPSHOT</act.version>
+        <act-morphia.version>0.4.1-SNAPSHOT</act-morphia.version>
+        <act-ebean.version>0.3.1-SNAPSHOT</act-ebean.version>
```
Comment (martin-g) [inline frameworks/Java/act/pom.xml]: Are those available at Maven central ?
I cannot find them.
And I don't see custom repositories in this file.

Comment (greenlaw110) [inline frameworks/Java/act/pom.xml]: there are snapshot repositories specified in the parent pom file:

```
    <parent>
        <groupId>org.sonatype.oss</groupId>
        <artifactId>oss-parent</artifactId>
        <version>7</version>
    </parent>
```

File: frameworks/Java/act/pom.xml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.actframework</groupId>
+            <artifactId>act-ebean</artifactId>
+            <version>${act-ebean.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>${h2.version}</version>
```
Comment (martin-g) [inline frameworks/Java/act/pom.xml]: This dependency is not used.

Comment (greenlaw110) [inline frameworks/Java/act/pom.xml]: Ah yes, I should remove them. Do you want me to submit a follow up PR or you can do it by yourself?

Comment (martin-g) [inline frameworks/Java/act/pom.xml]: I am just a contributor like you who is interested to see what others do :-)

Comment (greenlaw110) [inline frameworks/Java/act/pom.xml]: Ah I see, thanks for checking out my PR ;-)

File: .travis.yml [hunk 3]
```diff
@@ -10,185 +10,8 @@ python:
...
-    - "TESTDIR=Scala/fintrospect"
-    - "TESTDIR=Scala/play2-scala"
-    - "TESTDIR=Scala/scruffy"
-    - "TESTDIR=Scala/spray"
-    - "TESTDIR=Scala/s-server"
-    - "TESTDIR=Scala/http4s"
-    - "TESTDIR=Scala/finch"
-    - "TESTDIR=Ur/urweb"
+    - "TESTDIR=Java/act"
+
```
Comment (nbrady-techempower) [inline .travis.yml]: Make sure to correct the `.travis.yml` file. Note that if you're doing a lot of pushing and you want travis to only run for your framework, I've built a hook - in the commit message you can do `[ci fw-only Java/act]`

Comment (greenlaw110) [inline .travis.yml]: @nbrady-techempower Sorry this is my first time making contribution made some mistakes. I just rebased from master and resolved .travis.yml conflicts using the current master version

**Comment by greenlaw110** [comment]
@nbrady-techempower this should be the last push. I've already caught up latest change on Master branch. And run the CI on my travis account: https://travis-ci.org/greenlaw110/FrameworkBenchmarks/builds/197128994

Please let me know if there are any other issues you spot and need me to fix. Thanks... Green

**Comment by nbrady-techempower** [comment]
@greenlaw110 Thanks, this looks good to me.

ping @msmith-techempower for second look

**Comment by msmith-techempower** [comment]
Looks good; LGTM!

**Comment by greenlaw110** [comment]
Great! Thanks techempowers!

**Pull request merged**